src/test/java/blackjack/domain/ParticipantsTest.java

```diff
@@ -0,0 +1,40 @@
+package blackjack.domain;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ParticipantsTest {
+
+    @DisplayName("생성 테스트")
+    @Test
+    void create() {
+        assertThatCode(() -> new Participants(List.of("아톰", "구름")));
+    }
+
+    @DisplayName("참여자들은 딜러 포함 최소 2명 이상이어야 한다.")
+    @Test
+    void validatePlayerSize() {
+        assertThatThrownBy(() -> new Participants(List.of()))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
List.of() 는 java 라이브러리라서 큰 문제는 없겠지만, 보통은 어떤 로직이 예외를 던질수도 있다는 검증을 할 때 여러 메서드를 한 줄에 같이 두고 검증하지 않아요. 예를 들어 해당 테스트는 아래와 같이 분리해볼 수 있겠죠 : 
```suggestion
    @DisplayName("참여자들은 딜러 포함 최소 2명 이상이어야 한다.")
    @Test
    void validatePlayerSize() {
        List<String> emptyNames = Collections.emptyList();
        assertThatThrownBy(() -> new Participants(emptyNames))
                .isInstanceOf(IllegalArgumentException.class);
    }
```
그 이유가 무엇인지 생각해보죠!

Reply:
현재, 여러 메서드 호출을 한 줄에 두고 검증하지 않도록 개선했습니다. 좋은 피드백 감사합니다. 🙇🏻‍♂️

말씀 주신 방법에 대해서 두 가지 이유를 생각해봤습니다.

**첫 번째, 중요한 행위를 검증하는 것에 집중하기 어려울 수 있습니다. (신뢰)**

아래와 같은 상황에서 `Data.createPlayerNames`가 실패하여 아래 테스트가 성공할 수 있습니다.
`assertThatThrownBy` 내부에는 한 가지 행위만 독립적으로 있어야 신뢰할 수 있는 테스트를 만들 수 있습니다.

```java
 @DisplayName("참여자들은 딜러 포함 최소 2명 이상이어야 한다.")
 @Test
    void validatePlayerSize() {
        assertThatThrownBy(() -> new Participants(Data.createPlayerNames()))
                .isInstanceOf(IllegalArgumentException.class);
  }
```

**두 번째, 어떤 값이 왔을 때 잘못된 상황인지 다른 사람이 보기 유용합니다. (가독)**
```java
 @DisplayName("참여자들은 딜러 포함 최소 2명 이상이어야 한다.")
 @Test
    void validatePlayerSize() {
        List<String> emptyNames = Collections.emptyList();     
   
        assertThatThrownBy(() -> new Participants(emptyNames))
                .isInstanceOf(IllegalArgumentException.class);
  }
```

잘못된 값에 `emptyNames`라는 이름을 부여해, 검증 대상 메서드에 빈 리스트가 전달될 경우, 예외가 발생한다는 것을 더 명확하게 설명할 수 있습니다.

제가 놓치고 있는 부분이 있다면, 추가 피드백 부탁드립니다. : )

Reply:
정확합니다! 특히 첫번째 이유가 정말 중요하다고 생각이 드는데요, 나도 모르게 `내가 검증하고자하는 로직이 아닌 다른 로직` 이 테스트를 성공시켜버리는 상황이 발생할 수도 있어요. 그러면 이 테스트는 오히려 우리에게 잘못된 안정감을 주게 되고, 나중에 버그 발견할 때도 더 힘들게 만들 수 있어요. '여긴 테스트가 있으니 문제 없겠지' 하고 엉뚱한 곳을 확인하게 하니까요. 정말 잘 정리해주셨습니다!

Reply:
잘못된 안정감을 준다는 말이 인상 깊은 것 같아요. 앞으로도 이 부분 의식하면서 개발해보겠습니다. 감사합니다!! 🙇🏻‍♂️